doc/tech/openstack-release.md

```diff
@@ -9,7 +9,7 @@ The selection of a new OpenStack baseline release will be associated with a new
 
 ## 9.1 Triggering Events for next release selection
 This section specifies events that may trigger the selection of the next OpenStack release.
--	Complete change in architecture: OpenStack or OpenStack Service 
+-	Complete change in architecture: OpenStack, OpenStack Service or major change on an OpenStack mandatory service's API
```

Comment:
@karinesevilla can we modify "major change on an OpenStack mandatory service's API" to "major API change of an OpenStack RA-1 required service"

Reply:
@goyal done